### ISSUE TITLE
Tool flags a violation that is WCAG AAA - 'Links Open in New Window Without Warning'

### ISSUE THREAD

**Issue by clinkenb** [Padhma/research-guides-pa11y#3]
Perhaps this is something that merely needs to be corrected in the README, rather than the code. The current README states a scan for WCAG A and AA. 'Links Open in New Window Without Warning' seems to relate to [3.2.5 Change on Request, Level AAA](https://www.w3.org/WAI/WCAG21/Understanding/change-on-request.html) or [advisory technique G201](https://www.w3.org/WAI/WCAG21/Techniques/general/G201).

Thanks! I love the simplicity of this tool and its focus on LibGuides. Siteimprove's browser tool does not work well with our guides.
